packages/xdl/src/webpack-utils/WebpackCompiler.ts

```diff
@@ -93,7 +93,7 @@ export function printPreviewNotice(projectRoot: string, showInDevtools: boolean)
   );
 }
 
-export default function createWebpackCompiler({
+export function createWebpackCompiler({
```

Comment:
seems like a lot of this import refactor can be done in another PR without needing a review.

Reply:
I renamed this export so that I can re-export it in `internal` like this:
```ts
export * as WebpackCompiler from './webpack-utils/WebpackCompiler';
```
(I could have re-exported individual functions, but I didn't want to add `printInstructions`, `printSuccessMessages` etc. to `internal` without the `WebpackCompiler` namespace because otherwise it would be harder to see where they come from. I hope that makes sense.)